From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Wed, 5 May 2021 21:23:40 -0400
Subject: [PATCH] Add main() to volk_modtool script, usable as an entry point.

---
 python/volk_modtool/volk_modtool | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/python/volk_modtool/volk_modtool b/python/volk_modtool/volk_modtool
index 4f6f247..892d4a5 100755
--- a/python/volk_modtool/volk_modtool
+++ b/python/volk_modtool/volk_modtool
@@ -27,7 +27,7 @@ from optparse import OptionParser, OptionGroup
 import os
 import sys
 
-if __name__ == '__main__':
+def main():
     parser = OptionParser();
     actions = OptionGroup(parser, 'Actions');
     actions.add_option('-i', '--install', action='store_true',
@@ -126,3 +126,6 @@ if __name__ == '__main__':
             kernelset = my_modtool.get_current_kernels(base);
             for i in kernelset:
                 print(i);
+
+if __name__ == '__main__':
+    main()
-- 
2.34.1

